clayui.com/gatsby/createPages.js

```diff
@@ -33,15 +33,27 @@ const createDocs = (actions, edges, mdx, blacklist = []) => {
 		.forEach(
 			({
 				node: {
-					fields: {layout, redirect, redirectFrom, sibling, slug},
+					fields: {
+						layout,
+						redirect,
+						redirectFrom,
+						sibling,
+						siblings,
+						slug,
+					},
 				},
 			}) => {
 				const hasSibling = blacklist.find(
 					({
 						node: {
 							fields: {sibling},
 						},
-					}) => sibling === slug
+					}) => {
+						console.log('sibling: ', sibling);
+						console.log('slug: ', slug);
+
+						return sibling === slug;
+					}
```

Comment:
hey @kresimir-coko if I'm not mistaken we should treat `siblings` as a blacklist too, then we should add it in https://github.com/liferay/clay/blob/a50d9b724eedfab8fff5585d4a0cf783a7cb8740/clayui.com/gatsby/createPages.js#L183-L192

The logic of sibling is not to create a page for him here, because we only want his content that is taken from the docs template.